packages/replay/src/utils.ts

```diff
@@ -1,3 +1,9 @@
+// This module is meant to be somewhat browser-friendly.
```

Comment:
I tested the core of this PR with Cypress **without** changes to those utils here and it worked. So it might not be strictly needed but, for the time being, I feel more comfortable with adding this extra entrypoint for utils too.